### PR TITLE
Fixed playbook regexes

### PR DESCRIPTION
**Pull request by reutshal**
## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25480

## Description
Fixed playbook regexes to not start with the word `script`

### CODE REVIEW

File: demisto_sdk/commands/common/constants.py [hunk 1]
```diff
@@ -286,6 +286,7 @@
 PACKS_SCRIPT_README_REGEX = fr'{SCRIPT_DIR_REGEX}/README.md$'
 
 PACKS_SCRIPT_NON_SPLIT_BASE_REGEX = fr'{SCRIPTS_DIR_REGEX}/script-([^\\/]+)'
+PACKS_SCRIPT_TEST_PLAYBOOK = fr'{PACK_DIR_REGEX}\/{TEST_PLAYBOOKS_DIR}/script-([^\\/]+).yml$'
```
Comment (yaakovi) [inline demisto_sdk/commands/common/constants.py:289]: no need to escape slashes.
```suggestion
PACKS_SCRIPT_TEST_PLAYBOOK = fr'{PACK_DIR_REGEX}/{TEST_PLAYBOOKS_DIR}/script-([^\\/]+).yml$'
```

File: demisto_sdk/commands/common/constants.py [hunk 2]
```diff
@@ -338,7 +339,7 @@
 PLAYBOOK_README_REGEX = fr'{PLAYBOOK_BASE_REGEX}_README\.md$'
 
 TEST_SCRIPT_REGEX = r'{}{}.*script-.*\.yml$'.format(CAN_START_WITH_DOT_SLASH, TEST_PLAYBOOKS_DIR)
-TEST_PLAYBOOK_YML_REGEX = fr'{PACK_DIR_REGEX}\/{TEST_PLAYBOOKS_DIR}\/([^.]+)\.yml'
+TEST_PLAYBOOK_YML_REGEX = fr'{PACK_DIR_REGEX}\/{TEST_PLAYBOOKS_DIR}\/(?!script-)([^.]+)\.yml'
```
Comment (yaakovi) [inline demisto_sdk/commands/common/constants.py:342]:
```suggestion
TEST_PLAYBOOK_YML_REGEX = fr'{PACK_DIR_REGEX}/{TEST_PLAYBOOKS_DIR}\/(?!script-)([^.]+)\.yml'
```

File: demisto_sdk/commands/common/constants.py [hunk 3]
```diff
@@ -350,7 +351,7 @@
 
 PLAYBOOK_REGEX = r'{}(?!Test){}/playbook-.*\.yml$'.format(CAN_START_WITH_DOT_SLASH, PLAYBOOKS_DIR)
 
-TEST_PLAYBOOK_REGEX = r'{}{}/playbook-.*\.yml$'.format(CAN_START_WITH_DOT_SLASH, TEST_PLAYBOOKS_DIR)
+TEST_PLAYBOOK_REGEX = r'{}{}/(?!script-).*\.yml$'.format(CAN_START_WITH_DOT_SLASH, TEST_PLAYBOOKS_DIR)
```
Comment (yaakovi) [inline demisto_sdk/commands/common/constants.py:354]: we can delete this one once the TestPlaybooks folder is fully cleared...

File: demisto_sdk/commands/common/constants.py [hunk 4]
```diff
@@ -449,7 +450,8 @@
 
 YML_SCRIPT_REGEXES: List[str] = [
     PACKS_SCRIPT_YML_REGEX,
-    PACKS_SCRIPT_NON_SPLIT_YML_REGEX
+    PACKS_SCRIPT_NON_SPLIT_YML_REGEX,
+    PACKS_SCRIPT_TEST_PLAYBOOK
```
Comment (yaakovi) [inline demisto_sdk/commands/common/constants.py:454]: where do we use this?
do we want to validate those in the validate command?

File: demisto_sdk/commands/common/constants.py [hunk 5]
```diff
@@ -560,6 +562,7 @@
     PACKS_SCRIPT_PS_REGEX,
     PACKS_SCRIPT_TEST_PY_REGEX,
     PACKS_SCRIPT_README_REGEX,
+    PACKS_SCRIPT_TEST_PLAYBOOK,
```
Comment (yaakovi) [inline demisto_sdk/commands/common/constants.py:565]: same here

File: demisto_sdk/commands/common/constants.py (hunk 4 above)
Comment (reutshal) [reply demisto_sdk/commands/common/constants.py:454]: This is used in the schema validation.
Why wouldn't we want to validate the script used for test playbooks?

File: demisto_sdk/commands/common/constants.py (hunk 3 above)
Comment (reutshal) [reply demisto_sdk/commands/common/constants.py:354]: What do you mean? Is there any change planned?

Comment (yaakovi) [reply demisto_sdk/commands/common/constants.py:354]: this one is for TestPlaybook folder in the root of content (not inside packs)

File: demisto_sdk/commands/common/constants.py (hunk 4 above)
Comment (yaakovi) [reply demisto_sdk/commands/common/constants.py:454]: just making sure we are not running BC checks on those files

File: .circleci/config.yml [hunk 6]
```diff
@@ -56,6 +56,11 @@ jobs:
                 git checkout ${CONTENT_BRANCH_NAME}
               fi
 
+              if [[ $FAKE_MASTER = true ]]; then
+                demisto-sdk validate -a
+                exit 0
+              fi
+
               ./Tests/scripts/validate.sh
```
Comment (yaakovi) [inline .circleci/config.yml:64]:
```suggestion
              if [[ $FAKE_MASTER = true ]]; then
                  CIRCLE_BRANCH="master" ./Tests/scripts/validate.sh
              else
                  ./Tests/scripts/validate.sh
              fi
```

**Comment by reutshal** [comment]
Ran validate -a and it passed successfully

**Pull request merged**